app/src/main/java/com/sdp/swiftwallet/domain/model/CurrencyAdapter.java

```diff
@@ -59,6 +63,18 @@ public ViewHolder(@NonNull View itemView) {
             nameTextView = itemView.findViewById(R.id.idTextViewName);
             symbolTextView = itemView.findViewById(R.id.idTextViewSymbol);
             valueTextView = itemView.findViewById(R.id.idTextViewValue);
+
+            itemView.setOnClickListener(new View.OnClickListener(){
+
+                @Override
+                public void onClick(View view) {
+
+                    Intent intent = new Intent(context, CryptoGraphActivity.class);
+                    int pos = getAbsoluteAdapterPosition();
+                    intent.putExtra("currency", currencyArrayList.get(pos));
+                    context.startActivity(intent);
+                }
+            });
```

Comment:
This I guess you can do in a nicer way with a lambda

```suggestion
            itemView.setOnClickListener( ( view ) -> {
                 Intent intent = new Intent(context, CryptoGraphActivity.class);
                 int pos = getAbsoluteAdapterPosition();
                 intent.putExtra("currency", currencyArrayList.get(pos));
                 context.startActivity(intent);
            });
```